content/sensu-go/5.17/reference/secrets-providers.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Secrets providers"
+linkTitle: "Secrets Providers"
+description: "Sensu's secrets management capability allows you to avoid exposing secrets in your Sensu configuration. Read the reference to obtain secrets from one or more external secrets management providers and support references to external secrets in your Sensu configuration."
```

Comment:
```suggestion
description: "Sensu's secrets management capability allows you to avoid exposing secrets like usernames, passwords, and access keys in your Sensu configuration. Read the reference to obtain secrets from one or more external secrets providers and support references to external secrets in your Sensu configuration."
```